docs/user/openstack/affinity.md

```diff
@@ -0,0 +1,136 @@
+# How to set affinity rules for workers at install-time
+
+The Installer automatically creates Control plane nodes with a
+"soft-anti-affinity" policy. This means that each server will be created on a
+separate host, if enough hosts are available.
+
+The Installer does not set any affinity policy on Compute nodes; the choice is
+left to the user.
+
+We are going to create an OpenStack `server group` with an `anti-affinity`
+policy. Then we are going to modify the `MachineSet` definitions during the
+installation process, so that when `openshift-install create cluster` is run,
+the Compute nodes will directly be created with the given policy.
+
+### Step 1: create the Server group
+
+In OpenStack Compute (Nova), scheduler policies are set using Server groups.
+
+The first step is then to create the Server group.
+
+Depending on the OpenStack configuration, there are a maximum of four possible policies:
+* affinity
+* soft-affinity
+* anti-affinity
+* soft-anti-affinity
+
+These options are best described in the [OpenStack API reference][openstack-compute-api-docs].
```

Comment:
Is this a placeholder, or is there some docs wizardry to turn this into a real link?

Edit: it's sourcery. See bottom of document.